examples/mediapipe/BackgroundEnchantments/README.md

```diff
@@ -0,0 +1,39 @@
+# Background Enhancement Sample Application
+This sample application shows how to create background enhancement using [@vonage/ml-transformers](https://www.npmjs.com/package/@vonage/ml-transformers) package.
+# Demo
+You can see a demo of this sample running [here](https://vonage-background-enchantments-sample.s3.amazonaws.com/index.html)
+# Building the application
+**This is very simple**
```

Comment:
IMO this not add any info the build process so I would remove it.

Reply:
fixed